visualization/tslint.json

```diff
@@ -9,6 +9,7 @@
         "no-conditional-assignment": true,
         "new-parens": true,
         "no-unused-variable": [true, {"ignore-pattern": "^_"}],
+        "member-access": true,
```

Comment:
functions with no member-access are declared public now (they were public by default before anyway). Inform team to check files from time to time if some functions can be declared private

Reply:
See [wiki](https://github.com/MaibornWolff/codecharta/wiki/to-be-documented)